pkg/image/build.go

```diff
@@ -156,6 +167,26 @@ func Build(cfg *config.Config, dir, imageName string, secrets []string, noCache,
 		global.LabelNamespace + "has_init": "true",
 	}
 
+	if isUsingCogBaseImage {
+		labels[global.LabelNamespace+"using_cog_base_image"] = "true"
```

Comment:
Is there a reason we're doing underscores here and dashes the next line down?

Reply:
See also https://github.com/replicate/cog/pull/1747#discussion_r1649278794